kubernetes/zenko/charts/opal/values.yaml

```diff
@@ -0,0 +1,160 @@
+global:
+  orbit:
+    enabled: true
+    endpoint: https://api.zenko.io
+    pushEndpoint: https://push.api.zenko.io
+    workerMode: poll
+
+  locationConstraints: {}
+
+users: {}
+logging:
+  level: info
+
+mongodb:
+  replicaSet: rs0
+  replicas: 3
+
+kmip:
+  enabled: false
+  port: 5696
+  hosts: []
+  compoundCreate: false
+  bucketAttributeName: ''
+  pipelineDepth: 8
+
+proxy:
+  http: ''
+  https: ''
+  caCert: false
+  no_proxy: ''
+
+redis:
+  sentinel:
+    name: zenko
+  replicas: 3
+
+allowHealthchecksFrom: '0.0.0.0/0'
+
+blobserver:
+  endpoint: blob.zenko.local
+  enable_dns_storage_accounts: True
+  replicaCount: 3
+  replicaFactor: 1
+
+  asyncDeletes:
+    successfulJobsHistory: 1
+    failedJobsHistory: 3
+    jobs:
+      - name: container
+        command: ['yarn', 'cron:delete_container']
+        maxRetry: 1
+        maxConcurrentProcs: 10
+        maxConcurrentDeletes: 100
+        schedule: '*/5 * * * *'
+      - name: bucket
+        command: ['yarn', 'cron:delete_bucket']
+        maxRetry: 1
+        maxConcurrentProcs: 10
+        maxConcurrentDeletes: 100
+        schedule: '*/5 * * * *'
+      - name: overview
+        command: ['yarn', 'cron:delete_overview']
+        maxRetry: 1
+        maxConcurrentProcs: 10
+        maxConcurrentDeletes: 100
+        schedule: '*/5 * * * *'
+      - name: gc
+        command: ['yarn', 'cron:gc']
+        maxRetry: 1
+        maxConcurrentProcs: 10
+        maxConcurrentDeletes: 100
+        schedule: '*/5 * * * *'
+
+  image:
+    repository: registry.scality.com/sf-eng/blobserver
```

Comment:
We need to discuss the use of the repo `sf-eng` to be the one that we deliver.

I thought that it was supposed to be a playground for all of us to share and use images hosted on a registry, and now I see it here.

If you like the naming of it and want to use it that way, fine, but we need to define and advocate the fact that it is going to be used not only for dev purposes.

Otherwise I recommend that we create a new repo, in which we use for proper deliver, and continue to use `sf-eng` for CI and dev.

@tmacro @rahulreddy

Reply:
That makes sense to me as well, not sure if we want to change it for the 1.2 release or not

Reply:
I'm fine with this statement, I've created [ZENKO-2336](https://scality.atlassian.net/browse/ZENKO-2336) to create a discussion on the matter.

But still as `sf-eng` is a private repo how do we plan to deliver `blobserver`? If I don't have an answer to this question I'm not sure we can deliver 1.2

Reply:
So far its been implied that we would provide an `imagePullSecret` for those wanting to use blobserver